modules/basic/ds/hashmap.h

```diff
@@ -32,6 +32,14 @@ limitations under the License.
 #include "common/util/arrow.h"
 #include "common/util/uuid.h"
 
+#include "graph/fragment/property_graph_utils.h"
+#include "graph/thirdparty/BBHash/BooPHF.h"
```

Comment:
See how to include boost/leaf.h and libgrape-lite related files for how to process the third-party dependencies.

Reply:
Done